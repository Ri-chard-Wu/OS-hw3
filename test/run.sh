
cd ~/NachOS-4.0_MP3/code/build.linux
make clean
rm nachos
make



cd ~/NachOS-4.0_MP3/code/test
# cp /home/os2023/share/hw3t{1,2,3}.c ./
make clean
make hw3t1 hw3t2 hw3t3
../build.linux/nachos -e hw3t1 -e hw3t2 > run.out