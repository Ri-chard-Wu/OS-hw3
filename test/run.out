1
hw3t1
hw3t2
return value:1
2
return value:2
